### ISSUE TITLE
Allow templating of groups in timeline

### ISSUE THREAD

**Issue by ccollie** [almende/vis#529]
Similar to how templating works for timeline items, allow an optional groupTemplate option.

**Comment by josdejong** [comment]
Thanks for the suggestion, makes sense.

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
